.github/workflows/docker-build-startOs.yml

```
@@ -3,44 +3,105 @@ name: Build and Push Docker Images
-        push: true
-        tags: ${{ steps.meta.outputs.tags }}
-        labels: ${{ steps.meta.outputs.labels }}
-        cache-from: type=gha
-        cache-to: type=gha,mode=max
\ No newline at end of file
+      - name: Checkout repository
+        uses: actions/checkout@v4
+
+      - name: Set image tag for metadata
+        id: set_tag
+        run: |
```

actionlint (1.7.10)

[error] shellcheck reported issue in this script: SC2086:info:2:42: Double quote to prevent globbing and word splitting

(shellcheck)